config/routes.rb

```diff
@@ -3,4 +3,13 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  # resources :users do
+  #   resources :posts
+  # end
+
+  get 'users', to: 'users#index'
+  get 'users/:id', to: 'users#show'
+  get 'users/:user_id/posts', to: 'posts#index'
+  get 'users/:user_id/posts/:id', to: 'posts#show'
```

Comment:
- [ ] [OPTIONAL] You can use the following format to structure which is easier to maintain and modify:

```ruby
resources :users, only: [:index, :show] do
   resources :posts, only: [:index, :show]
end
```